curl.mustache

```diff
@@ -1,7 +1,7 @@
 {{#request.name}}## {{{request.name}}}
 {{/request.name}}{{#request.cURLDescription}}
 {{{request.cURLDescription}}}
-{{/request.cURLDescription}}curl -X "{{{request.method}}}" "{{{request.url}}}" \
+{{/request.cURLDescription}}curl {{#request_is_head}}--head {{/request_is_head}}-X "{{{request.method}}}" "{{{request.url}}}" \
```

Comment:
How about:

```
{{/request.cURLDescription}}curl {{#request_is_head}}--head {{/request_is_head}}{{#specify_method}}-X "{{{request.method}}}" {{/specify_method}}"{{{request.url}}}" \
```

And define:

```js
"specify_method": request.method != "GET" && request.method != "HEAD"
```

So it's not too redundant with `--head` and also we take the chance to add a tiny optimization not specifying `GET` method when it's the default.